R/drive_publish.R

```diff
@@ -1,9 +1,15 @@
-#' Publish Google Drive file.
+#' Publish Google Drive file
```

Comment:
I see. But I don't think that's clear to most people. I think we can suppress it in places like titles. As commented elsewhere, we need to pick some vocabulary re: Docs, Sheets, etc. and then use it consistently. Yeah in the description, in this case.